### ISSUE TITLE
Feature Request: Manual Barcode Entry

### ISSUE THREAD

**Issue by ArchEngel0** [MalcolmMielle/Fridgify#4]
Just came across a barcode that I couldn't scan (with any barcode scanner, not just yours) but the number was plainly visible. It would be great if users could manually enter a barcode in case of situations like these.

**Comment by MalcolmMielle** [comment]
Just trying to understand what would be the main use of having this capability. The problem I see is that you still wouldn't be able to scan it the next time you come across the product anyway? Is it for the integration with openfoodfacts ?

**Comment by ArchEngel0** [comment]
Yep, just for integration - I believe this barcode may have just been printed funny, but there is a possibility it is just wrong. Regardless, this would be a very low priority feature request as I think it's use case is quite small. Thanks for responding!
